.travis.yml

```diff
@@ -25,7 +25,7 @@ script:
     - ./run_checkdocs.py
     - ./run_unittests.sh
     - ./run_pylint.sh --FF
-    - inspekt style --disable E501,E265,W601,E402,E731
+    - inspekt style --disable E501,E265,W601,E402,E731,E221
```

Comment:
Nono, doing it commit by commit AND the whole shebang at the same time :dancer: 

Grrr, stooped pep8.  I have half-a-mind to just nuke the damn thing.